File: C/section06/S06E05.c
#include <stdio.h>
#include <ctype.h>

int main() {
    float totalFishCaughtWeight, totalWeightOverLimit;
    int fineToPay = 0;
    const int FINE_VALUE = 4;

    printf("Please type the total fish caught weight value: \n");
    scanf("%f", &totalFishCaughtWeight);

    if (totalFishCaughtWeight > 50) {
        totalWeightOverLimit = totalFishCaughtWeight - 50;
        fineToPay = ((int) totalFishCaughtWeight - 50) * FINE_VALUE;
    }

    printf("\nTotal Fish Caught Weight: %.2f", totalFishCaughtWeight);
    printf("\nTotal Weight Overlimit: %.2f", totalWeightOverLimit);
    printf("\nTotal Fine to pay: R$ %d,00", fineToPay);

    return 0;
}

File: Python/section06/S06e05.py
p = float(input ("Please enter fish weight: "))
e = 0
m = 0
mValue = 4

if (p - 50 > 0):
    e = p - 50
    m = e * mValue

print("\nFish caught weight: {0:.2f}\nWeight overlimit: {1:.2f}\nFine value: R${2:.2f}".format(p,e,m))